src/controllers/resources/resourceController.ts

```
@@ -0,0 +1,334 @@
+        req.body.parameters = JSON.parse(req.body.parameters);
+      } catch (e) {
+        return next(new BadRequestError('Invalid parameters JSON'));
+      }
+    }
+
+    const validatedRequestBody = CreateResourceRequestSchema.parse(req.body);
+    const { resourceType, name } = validatedRequestBody;
+
+    let result;
+
+    const resourceParams = (validatedRequestBody as any).parameters;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)